plugins/source/pagerduty/codegen/recipes/base.go

```diff
@@ -0,0 +1,353 @@
+package recipes
+
+import (
+	"bytes"
+	"embed"
+	"errors"
+	"fmt"
+	"go/format"
+	"os"
+	"path"
+	"reflect"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/cloudquery/plugin-sdk/caser"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/iancoleman/strcase"
+)
+
+type Resource struct {
+	SubService            string
+	Struct                interface{}
```

Comment:
use `any`

Reply:
`interface{}` is fine - most of our codebase uses that.. (including the code I copy-pasted this from)